### PR TITLE
Videos page

### PR DESCRIPTION
**Pull request by saicharan-gouru**
-Added videos
-Created video listing page
-Rendered videos

Screenshot (208)
Screenshot (209)

### CODE REVIEW

File: src/pages/Videoslisting/Videolisting.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    text-decoration: none;
+    color: var(--black);
+    font-weight: bold;
+    padding: 1rem;
+    cursor: pointer;
+}
+
+.thumbnail {
+    width: 100%;
+    height: 250px;
```
Comment (Logan1x) [inline src/pages/Videoslisting/Videolisting.css:27]: Could have used rem. it varies when you vary the screen size of the device.

**Review by Logan1x — APPROVED** [review]
Read more about meta tags and og tags. That can be improved in the end. You can proceed with the merge.

File: src/pages/Videoslisting/Videolisting.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.avatar {
+    width: 40px;
+    height: 40px;
```
Comment (omkarpatke) [inline src/pages/Videoslisting/Videolisting.css:32]: suggestion: use rem if the value is greater than 16px.

File: src/pages/Videoslisting/Videolisting.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    margin: 1rem;
+}
+
+.duration {
+    background-color: var(--primary-color);
+    color: var(--white);
+    padding: 10px;
+    font-weight: bold;
+    position: absolute;
+    margin-left: -3rem;
```
Comment (omkarpatke) [inline src/pages/Videoslisting/Videolisting.css:52]: suggestion: avoid -ve margin to maintain consistency.

**Review by omkarpatke — CHANGES_REQUESTED** [review]
UI looks good👍
code is also good readable,
suggestion: 
 - In UI add more vertical gap on videos. 
 - do suggested changes on CSS.
 then feel free to merge.

**Pull request merged**